py3status/modules/nvidia_temp.py

```diff
@@ -61,17 +62,25 @@ def nvidia_temp(self):
             return {
                 'cached_until': self.py3.CACHE_FOREVER,
                 'color': self.py3.COLOR_BAD,
-                'full_text': STRING_UNAVAILABLE
+                'full_text': STRING_NOT_INSTALLED
             }
 
-        temps = self.py3.command_output('nvidia-smi -q -d TEMPERATURE')
-        temps = set(TEMP_RE.findall(temps))
-        if temps == []:
+        try:
+            temps = self.py3.command_output('nvidia-smi -q -d TEMPERATURE')
+            temps = set(TEMP_RE.findall(temps))
```

Comment:
We only need the `command_output()` bit in the try.  If we had a problem with the findall() etc it would be better to see that error.